Updated levels to match current retail

kaincenteno opened the pull request:
<!-- place 'x' mark between square [] brackets to affirm: -->
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](https://github.com/project-topaz/topaz/blob/master/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

kaincenteno committed:
Updated levels to match current retail
fixed some missions with "is mission" that were over cap

TeoTwawki commented:
https://github.com/project-topaz/topaz/blob/master/conf/map.conf#L170
didn't need updated, tried to tell you in discord

kaincenteno commented:
changed everything that has ismission = 1 to cap at 75 (as some of them were higher than that level)
reverted all changes to anything level 75 and below
